package.json

```diff
@@ -57,9 +57,9 @@
     "docker-parse-image": "^3.0.1",
     "js-yaml": "^4.1.0",
     "lodash": "^4.17.21",
-    "screwdriver-config-parser": "^10.4.3",
-    "screwdriver-data-schema": "^23.5.1",
+    "screwdriver-config-parser": "file:../config-parser/screwdriver-config-parser-10.0.0.tgz",
+    "screwdriver-data-schema": "file:../data-schema/screwdriver-data-schema-22.0.0.tgz",
     "screwdriver-logger": "^2.0.0",
-    "screwdriver-workflow-parser": "^4.3.0"
+    "screwdriver-workflow-parser": "file:../workflow-parser/screwdriver-workflow-parser-4.0.0.tgz"
```

Comment:
TODO: Update the version after https://github.com/screwdriver-cd/workflow-parser/pull/55 is merged